### PR TITLE
.: adding travis file for future CI

### PR DESCRIPTION
**Pull request by vbatts**
https://github.com/opencontainers/specs/issues/90

Signed-off-by: Vincent Batts <vbatts@redhat.com>

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: go
+go:
+  - 1.4.2
+  - 1.3.3
+
+sudo: false
+
+before_install:
+  - go get code.google.com/p/go.tools/cmd/vet
```
Comment (LK4D4) [inline .travis.yml]: I'm pretty sure that it should be in `golang.org/x` now

Comment (vbatts) [inline .travis.yml]: fixed

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**